include/rocksdb/perf_level_by_bitfield.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+#include <stdint.h>
+
+namespace rocksdb {
+
+struct PerfLevelByBitField {
```

Comment:
Please think about a better name for it. BidField is an implementation and we should avoid that in the name.

Reply:
How about calling it PerfFlags? Since we are deprecating levels by using flags.